src/common/components/Header/index.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { withRouter } from 'react-router-dom';
 
-import { Layout, Navbar } from '../';
+import ResponsiveNavMenu from '../../../packages/responsive-nav-menu';
 
-const Header = ({ children, history, location }) => (
+import { Layout } from '../';
+
+const Header = ({ children }) => (
   <Layout.Header>
-    <Navbar activePathname={location.pathname} history={history}>
-      {children}
-    </Navbar>
+    <ResponsiveNavMenu>{children}</ResponsiveNavMenu>
   </Layout.Header>
 );
 
 Header.propTypes = {
   children: PropTypes.func.isRequired,
```

Comment:
if children is not being called here it seems weird that its function

Reply:
yes